src/components/AddressSearch/index.js

```diff
@@ -140,24 +140,46 @@ const defaultProps = {
 // Do not convert to class component! It's been tried before and presents more challenges than it's worth.
 // Relevant thread: https://expensify.slack.com/archives/C03TQ48KC/p1634088400387400
 // Reference: https://github.com/FaridSafi/react-native-google-places-autocomplete/issues/609#issuecomment-886133839
-function AddressSearch(props) {
+function AddressSearch({
+    canUseCurrentLocation,
+    containerStyles,
+    defaultValue,
+    errorText,
+    hint,
+    innerRef,
+    inputID,
+    isLimitedToUSA,
+    label,
+    maxInputLength,
+    network,
+    onBlur,
+    onInputChange,
+    onPress,
+    predefinedPlaces,
+    preferredLocale,
+    renamedInputKeys,
+    resultTypes,
+    shouldSaveDraft,
+    translate,
+    value,
+}) {
```

Comment:
I had to spread props to avoid disabling eslint exhaustive-deps rule in line 372.